File: fovus/aws/lambda/fovus_bucket_export.js
import { DynamoDBDocumentClient, PutCommand } from "@aws-sdk/lib-dynamodb";
import { DynamoDBClient } from "@aws-sdk/client-dynamodb";

const client = new DynamoDBClient({});
const docClient = DynamoDBDocumentClient.from(client);

export const handler = async (event) => {
    try {
        // Define the DynamoDB table name
        const tableName = 'fovus_ddb';
        
        // Define the put parameters
        const params = {
            TableName: tableName,
            Item: event
        };
        
        // Put the event data into the DynamoDB table
        await docClient.send(new PutCommand(params));
        
        // If successful, return a success response
        return {
            statusCode: 200,
            body: 'Event added to DynamoDB successfully'
        };
    } catch (error) {
        // If any error occurs, return an error response
        return {
            statusCode: 500,
            body: `Error adding event to DynamoDB: ${error.message}`
        };
    }
};

File: fovus/aws/lambda/fovus_bucket_export.py
import json
import boto3

dynamodb = boto3.resource('dynamodb')

def lambda_handler(event, context):
    try:
        # Get the DynamoDB table reference
        table = dynamodb.Table('fovus_ddb')
        
        # Put the event data into DynamoDB table
        response = table.put_item(
            Item=event
        )
        
        # If successful, return a success response
        return {
            'statusCode': 200,
            'body': 'Event added to DynamoDB successfully'
        }
    except Exception as e:
        # If any error occurs, return an error response
        return {
            'statusCode': 500,
            'body': f'Error adding event to DynamoDB: {str(e)}'
        }